skglm/datafits/single_task.py

```diff
@@ -423,9 +423,9 @@ def intercept_update_step(self, y, Xw):
 class Poisson(BaseDatafit):
     r"""Poisson datafit.
 
-    The datafit reads::
+    The datafit reads:
 
-    (1 / n_samples) * \sum_i (exp(Xw_i) - y_i * Xw_i)
+    .. math:: \frac{1}{\texttt{n_samples}} \sum_i (\exp(Xw_i) - y_i Xw_i)
```

Comment:
Xw_i migh be ambiguous. Using (Xw)_i would be preferrable.
While we're at it, using \sum_{i=1}^{n_\mathrm{n_samples}} would be more explicit too